_posts/2022-09-05-edge-example-app-cpu-temp.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title:  "Edge Example App: CPU Temperature"
+date:   2022-09-05 03:27:05 +0100
+categories: flotta
+author: Ahmad Ateya <ahmad.m.ateya@gmail.com>
+tags:
+- flotta
+- devices
+- workloads
+
+summary-1:
+---
+Edge Example App is an app for Flotta Edge devices, with a workload that will be deployed on the device that has 2 main features:
+
+Sensing the Internet (which helps to construct devices network topology). (see [part 1]() and [part 2]())
+Read CPU temperature (which indicate how much load the device is handling).<= this article
+
+As Project Flotta goal is to manage workloads deployed on small-footprint devices, reading the CPU temperature is a good way to know how much load the device is handling. this App Along with the Web UI Interface, helps to collect and visualize the CPU temperature of the device over the day by presenting it into Line Graph, making it easy to identify the time of the day when the device is handling more load.
+
+
+## How this app works
+
+This app reads the CPU temperature from the device and saves it to a file on the device, the device worker will send it to the edge cluster, and the edge cluster will save it to the S3 bucket.
+
+in the backend side, the edge cluster will read the file from the S3 bucket, calculate the average temperature for every hour and expose this data to the web UI with endpoints, which will present it to the user into line graph.
```

Comment:
`s/in/In/

not sure: "into line graph" or "as line graph"? can you check?

Reply:
as a line graph is better (Y)